### PR TITLE
Validations

### PR DESCRIPTION
**Pull request by AngelRRuizM**

### CODE REVIEW

File: server/controllers/tutoring.js [hunk 1]
```diff
@@ -2,198 +2,25 @@ const Tutoring = require('../models').Tutoring;
...
-                        code: 10
-                    }
-                });
+        return new Tutoring(tutoring)
+        .save()
+        .then((postedTutoring) => {
+            res.status(200).send(postedTutoring);
+        })
+        .catch((err) => {
+            res.status(500).send({
```
Comment (Syul968) [inline server/controllers/tutoring.js:16]: We can use our new ErrorFactory here!

**Review by Syul968 — APPROVED** [review]
🐱

**Comment by AngelRRuizM** [comment]
@Syul968 !!!!!!!!!!!!!!!!!

File: test/tutoring.spec.js [hunk 2]
```diff
@@ -12,23 +9,8 @@ chai.use(chaiHttp);
...
-        userID: '5d8d49a56ee837016abcd2aa',
-        startTime: "12:00",
-        endTime: "23:00",
-        notes: 'It\'s not the best student',
-        paymentMethod: 'cash'
-    };
     let attributeNames = ['date', 'lat','long','locationType', 'locationName', 'topicID', 'tutorID', 'userID','startTime', 'endTime', 'notes', 'paymentMethod']; 
-    let attributesValue = [];
-    it('shouldCreate a tutoring', (done) => {
+    /*it('shouldCreate a tutoring', (done) => {
```
Comment (LDanielMedinaC) [inline test/tutoring.spec.js:13]: Please delete comments before pushing anythings

**Review by LDanielMedinaC — APPROVED** [review]

**Pull request merged**